tools/migration/cometv1/complex.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
so this dir could just be called v54 and include the comet and other import changes in 1